locales/es/criteria.json

```diff
@@ -45,33 +45,33 @@
   "criteria.applicant-relatonships.label": "",
   "criteria.applicant_lost_a_family_member.label": "¿Perdió recientemente a un ser querido?",
   "criteria.applicant_lost_a_family_member.values1": "sí",
-  "criteria.applicant_lost_a_family_member.values2": " no",
+  "criteria.applicant_lost_a_family_member.values2": "no",
   "criteria.applicant_lost_a_service_member.label": "¿Perdió recientemente a un ser querido que prestaba servicio en el ejército?",
   "criteria.applicant_lost_a_service_member.values1": "sí",
-  "criteria.applicant_lost_a_service_member.values2": " no",
+  "criteria.applicant_lost_a_service_member.values2": "no",
   "criteria.applicant_child_spouse_SS.label": "Usted es hijo o cónyuge de una persona que recibe beneficios de jubilación o por discapacidad.",
   "criteria.applicant_family_of_service_member.label": "Usted es familiar de una persona que prestó servicio activo en el ejército, la Marina o la Fuerza Aérea y quien",
   "criteria.applicant_family_of_service_member.values1": "desapareció en acción",
-  "criteria.applicant_family_of_service_member.values2": " fue capturada o detenida en cumplimiento del deber",
-  "criteria.applicant_family_of_service_member.values3": " sufrió discapacidad a raíz del servicio prestado",
+  "criteria.applicant_family_of_service_member.values2": "fue capturada o detenida en cumplimiento del deber",
+  "criteria.applicant_family_of_service_member.values3": "sufrió discapacidad a raíz del servicio prestado",
   "criteria.applicant_care_for_child.label": "Usted cuida del hijo/hija de la persona que falleció, se retiró o tiene discapacidades y el hijo/hija tiene menos de 16 años o una discapacidad.",
   "criteria.applicant_relationship.label": "Su relación con la persona que falleció es",
   "criteria.applicant_relationship.values1": "cónyuge",
-  "criteria.applicant_relationship.values2": " hijo/hija",
-  "criteria.applicant_relationship.values3": " padre/madre",
-  "criteria.applicant_relationship.values4": " otro miembro de la familia",
-  "criteria.applicant_relationship.values5": " representante personal u oficial",
+  "criteria.applicant_relationship.values2": "hijo/hija",
+  "criteria.applicant_relationship.values3": "padre/madre",
+  "criteria.applicant_relationship.values4": "otro miembro de la familia",
+  "criteria.applicant_relationship.values5": "representante personal u oficial",
   "criteria.applicant_paid_funeral_expenses.label": "Usted pagó los gastos de funeral/entierro.",
   "criteria.applicant_funeral_reimbursment.label": "NO fue reembolsado el costo del funeral/entierro por ninguna organización ni agencia del gobierno.",
```

Comment:
🤔 Not sure we need to capitalize NO.

Reply:
@fpigeonjr Thank you for the review!! "NOT" is capitalized in English version, and that's why Spanish translation "NO" is also capitalized.